### ISSUE TITLE
Can't drag ANY content (text ,image ,url) from Safari to Typora

### ISSUE THREAD

**Issue by kls1991** [typora/typora-issues#1073]
OS :  High Sierra

Issue : Can't drag ANY content (text ,image ,url) from Safari to Typora,Chrome neither.Copy an image(not url) in browser and paste to Typora work fine.

Is Typora surpposed to be like this ? Or I do something wrong ?

I can't get barely any information from google , help please.

Sorry for my bad English.

**Comment by vassudanagunta** [comment]
Copying from #899 so that details of desired URL dragging are not lost:

Currently if you drag a URL from Safari onto a Typora page, nothing happens (It is ignored). The behavior is the same whether I drag the URL from the address bar, or drag a link from web content.

TextEdit (in Rich Text mode) or other Markdown editors such ByWord behave as expected:
- Dragging the URL from the address bar inserts into Typora a link to the URL whose anchor text is the title of the webpage.
- Dragging a link from the web content inserts into Typora an identical link (same anchor text and URL).

**Comment by vassudanagunta** [comment]
If an image is dragged over, it would be nice if a dialog popped up and asked: "Link to remote image or copy image to local folder?"

**Comment by abnerlee** [comment]
Should be fixed in new release

**Comment by vassudanagunta** [comment]
@abnerlee The *Link / URL Drag & Drop* works like a charm! Thank you!

But the *Image Drag & Drop* only links to the remote image, with no option either via dialog or meta-key to copy the image to the local folder, respecting Typora's image insert setting.

**Comment by vassudanagunta** [comment]
Yes, I have `Copy image to current folder` and `Apply above rules for online images` selected. 

I just repeated a test many times, and each time I got the remote URL. But a couple of times I got `![img](blob:file:///269c09f1-fe46-49fe-a196-346347c43e69)`, with no image display.  I haven't been able to repeat. I am using Safari Technology Preview. I can't test with regular Safari at the moment.
